packages/salesforcedx-vscode-core/src/channels/channelService.ts

```diff
@@ -64,14 +64,14 @@ export class ChannelService {
 
       this.channel.append(' ');
       if (data !== undefined) {
-        this.channel.appendLine(
-          nls.localize('channel_end_with_error', data.message)
-        );
-
         if (/sfdx.*ENOENT/.test(data.message)) {
           this.channel.appendLine(
             nls.localize('channel_end_with_sfdx_not_found')
           );
+        } else {
```

Comment:
Since we're removing the CLI installed checks during startup I'm also modifying this so that is we receive a `ENOENT` message from attempting to run a CLI command we only print the message on how to install/configure it instead of also printing the error message from the CLI.